vllm/entrypoints/openai/api_server.py

```diff
@@ -165,6 +167,36 @@ async def create_embedding(request: EmbeddingRequest, raw_request: Request):
         return JSONResponse(content=generator.model_dump())
 
 
+@router.post("/v1/load_lora_adapter")
+async def load_lora_adapter(request: LoadLoraAdapterRequest):
```

Comment:
we should allow operators to disable those methods and have them disabled by default; having user specify the path to the adapter themselves is inherently unsafe.

Reply:
makes sense. let me add some feature gate for it

Reply:
I added a feature flag now and leave a TODO here.  I think once it's enabled, it's still has the risk. The key problem is if we want to support admin operations, then we have to distinguish the users. This is something not supported yet. In our system, we distinguish the user at proxy level. If the vLLM is public to users directly, then that's a problem